sidebars.js

```diff
@@ -36,17 +31,41 @@ module.exports = {
         'guides/publishing',
       ],
     },
-  ],
-  comlinkReferenceSidebar: [
     {
       type: 'category',
-      label: 'Comlink References',
-      collapsed: false,
+      label: 'Working with providers',
+      collapsed: true,
       items: [
-        { type: 'doc', id: 'comlink/reference/profile', label: 'Profile' },
-        { type: 'doc', id: 'comlink/reference/map', label: 'Map' },
-        { type: 'doc', id: 'comlink/reference/provider', label: 'Provider' },
+        'guides/find-provider-by-name',
+        'guides/add-new-provider',
+        'guides/using-multiple-providers',
       ],
     },
+    'guides/api-keys',
+    'guides/integrations-monitoring',
+  ],
+};
+
+const comlink = {
```

Comment:
I left it kind of mid way, because I didn’t want to have reference separated at first, but then it made sense. And since I didn’t add sidebar for reference, I will and move Comlink there.